cmd/update/cluster/runner.go

```diff
@@ -108,10 +108,7 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 				},
 			},
 		}
-		messageFormat := `An upgrade of cluster %s to release %s has been scheduled for
-
-		%v, (%v)
-		`
+		messageFormat := "An upgrade of cluster %s to release %s has been scheduled for\n    %v, (%v)"
```

Comment:
```suggestion
		messageFormat := "An upgrade of cluster %s to release %s has been scheduled for\n\n    %v, (%v)"
```

Sorry, forgot one newline